pkg/manager/manager.go

```diff
@@ -43,6 +43,7 @@ type Manager struct {
 	OperationalStateChannel chan events.OperationalStateEvent
 	SouthboundErrorChan     chan events.DeviceResponse
 	Dispatcher              dispatcher.Dispatcher
+	OperationalStateCache   map[topocache.ID]map[string]*change.TypedValue
```

Comment:
I thought you were having 2 separate caches per device - one operational, one state

Reply:
no, everything is put in the same cache, which should actually be renamed as read only cache.
is it a problem to have just one ?

Reply:
It's a gRPC thing really - If we merge them together here then we can't ever separate them